organization/external-communication.md

```diff
@@ -0,0 +1,168 @@
+* Re-posts of interesting pyOpenSci blog posts
+
+These posts are all structured as blog posts, using appropriate heading and subheading formats within LinkedIn. In fact, all of our newsletters are first published on the pyOpenSci blog, and should follow the same tone and format.
+
+When a newsletter post is a re-post from the pyOpenSci blog, it's important to include text and a link back to the original blog post, indicating where the post was originally published. This prevents link cannibalisation, where different links with the same content compete for keyword rankings. We want all keyword rankings to be directed back to the pyOpenSci website wherever possible.
+
+### Newsletter success metrics
+
+Because pyOpenSci is still in the early days of its newsletter, our success metrics are focused primarily on growth and engagement rate. We are currently aiming for:
+
+* 1.0% (or higher) monthly increase in newsletter subscribers
+* 3.0% (or higher) average monthly engagement rates
+
+## YouTube
+
+The [pyOpenSci YouTube channel](https://youtube.com/@pyopensci) is still under development, and currently only has one published video. While we are still in the process of developing a YouTube strategy, the best practices for YouTube videos include the following:
+* Provide accurate captions
+* Create an eye-catching thumbnail for each video, using a similar design aesthetic for videos that occur in a series
+* Use timestamps in the video description
+* Promote the video on socials within the first 20 minutes of the video launching
+* Use no more than two hashtags per video
+
+At this point in time, there are no active goals or success metrics for the YouTube channel. We'll be sure to update this section once we're regularly updating our channel!
+
+## Discourse
+
+pyOpenSci also maintains a [public Discourse forum](https://pyopensci.discourse.group/). Unlike Slack, which has restricted access, anyone is able to join and participate in Discourse. Discourse is a place to announce new packages in our ecosystem, to post memos to the community, as well as to announce meetings and other events. We also want the community to use it to ask questions around peer review and packaging.
+
+pyOpenSci is currently in the process of sunsetting Discourse as it has not proven to be a widely-used platform for our community. We are currently exploring the use of GitHub discussions.
+
+## Zenodo
+
+Zenodo is a general-purpose research repository where any research output can be shared by its users, where pyOpenSci maintains a [community group](https://zenodo.org/communities/pyopensci?q=&l=list&p=1&s=10&sort=newest). When pyOpenSci staff give a presentation, they will update the community group with their talk materials. In addition, when community members give presentations, they have the option of adding them to the pyOpenSci Zenodo community group.
```

Comment:
A few important things to add

1. pyOpenSci uses Zenodo as a way to provide citable entities for its GitHub org repos.  (it provides DOI's but they aren't cross-ref doi but still citable). we use it for most of our repos and add the citation badge to the readme of each repo. then we add the badge to the home page of content that is being tracked in zenodo. 

When there are major updates to content such as the packaging guide, we create a new release and zenodo provides and updated DOI associated with those commits. .

we also have a zenodo communiy  that people can add their presentations, blog posts, or any content really to. This allows the community member (or pyopensci staff) to gain a citable DOI for the element deposited in the community but also they still own the addition. 

the community is moderated by pyOS staff admins. So anyone can suggest and addition but we moderate what is added.